commerce-order-web/src/main/resources/META-INF/resources/order_shipment/shipping_summary.jsp

```diff
@@ -24,7 +30,9 @@
 			<commerce-ui:info-box
 				title="courier-detail"
 			>
-				Shipping ID <clay:link href="#" label="1ZKAS23JLKNASDJL" />
+				<liferay-ui:message key="shipping-id" /> <clay:link href="#" label="<%= commerceShipment.getTrackingNumber() %>" />
```

Comment:
This should really be a lang key with an argument: `shipping-id-x`. That way if this statement makes more sense in a different order in another language, the translator has the power to do that. (It should be fine to pass the `clay:link` as the argument so that we still have the ID linked.) Appending something to a translated value is almost always the wrong way to do it.